compiler-plugin/src/main/kotlin/arrow/meta/quotes/expression/TryExpression.kt

```diff
@@ -0,0 +1,44 @@
+package arrow.meta.quotes.expression
+
+import arrow.meta.quotes.Scope
+import arrow.meta.quotes.ScopedList
+import org.jetbrains.kotlin.psi.KtBlockExpression
+import org.jetbrains.kotlin.psi.KtCatchClause
+import org.jetbrains.kotlin.psi.KtFinallySection
+import org.jetbrains.kotlin.psi.KtTryExpression
+
+/**
+ * <code>""" try $tryBlock $catch $finally """.tryExpression</code>
+ *
+ * A template destructuring [Scope] for a [KtTryExpression].
+ *
+ * ```kotlin:ank:silent
+ * import arrow.meta.Meta
+ * import arrow.meta.Plugin
+ * import arrow.meta.invoke
+ * import arrow.meta.quotes.Transform
+ * import arrow.meta.quotes.tryExpression
+ *
+ * val Meta.reformatTry: Plugin
+ *  get() =
+ *   "ReformatTry" {
+ *    meta(
+ *     tryExpression({ true }) { e ->
+ *      Transform.replace(
+ *       replacing = e,
+ *       newDeclaration = """ try { $tryBlock } $catch finally {  $finally } """.`try`
+ *      )
+ *      }
+ *     )
+ *    }
+ * ```
+ */
+class TryExpression(
+  override val value: KtTryExpression?,
+  val tryBlock: Scope<KtBlockExpression> = Scope(value?.tryBlock),
+  val catchClauses: ScopedList<KtCatchClause> = ScopedList(
+    prefix = "catch ",
+    value = value?.catchClauses ?: listOf()
```

Comment:
This was the other side of when formatting lists doesn't go quite right

Reply:
You can also use `orEmpty()` in place of `?: listOf()`

Reply:
Thanks! I'll try that!